app/models/facility_group.rb

```diff
@@ -11,6 +11,8 @@ class FacilityGroup < ApplicationRecord
   has_many :medical_histories, through: :patients
   has_many :communications, through: :appointments
 
+  belongs_to :protocol
```

Comment:
We should remember to move this to the admin notion of facility groups, when we separate those concerns.